profiling/perfetto-connector/CMakeLists.txt

```diff
@@ -1,3 +1,14 @@
-kp_add_library(kp_perfetto_connector libperfetto-connector.cpp perfetto/perfetto.cc)
-target_include_directories(kp_perfetto_connector PRIVATE perfetto)
+# perfetto/perfetto.cc is third-party code, so we want to disable all warnings for it
+add_library(perfetto_static STATIC perfetto/perfetto.cc)
+set_target_properties(perfetto_static PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Comment:
Why do we built the library explicitly as `static` with `fpic` instead of shared?

Reply:
I figured it was less opportunity for something to go wrong during the build. I don't have a strong opinion.